src/collections/_documentation/platforms/android/index.md

```diff
@@ -0,0 +1,597 @@
+            // Add a callback that will be used before the event is sent to Sentry.
+            // With this callback, you can modify the event or, when returning null, also discard the event.
+            options.setBeforeSend((event, hint) -> {
+                if (SentryLevel.DEBUG.equals(event.getLevel()))
+                    return null;
+                else
+                    return event;
+            });
+        });
+    }
+}
+```
+
+For more details about the available configurations, see [Configuration options](#configuration-options).
+
+## Capturing Errors
+
+### Crash Reporting
+
+Your application will crash whenever a thrown exception goes uncaught. The Sentry SDK catches the exception right before the crash and builds a crash report that will persist to the disk. The SDK will try to send the report right after the crash, but since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+If there is a fatal error in your native code, the process is the same. The difference is, a restart is always needed, as a hard crash immediately stops the app.
+
+The NDK is not only catching the unhandled exceptions but is also set as a signal handler to be able to react to the signals from OS. When the application is about to crash, an error report is created and saved to disk. The SDK will try to send the report right after the crash, but since the environment may be unstable at the crash time, the report is guaranteed to send once the application is started again.
+
+### Capturing Errors manually
+
+Insight into the health of your application doesn't require crashes. You can report exceptions and messages manually using our API.
+
+```java
+public class DemoClass {
+    /**
+        * An example method that throws an exception.
+        */
+    public void unsafeMethod() {
+        throw new RuntimeException();
+    }
+
+    /**
+        * An Example of how to report an error and how to enrich the error context.
+        */
+    void logWithStaticAPI() {
+    /*
+    Record a breadcrumb in the current context, which will be sent
+    with the next event(s). By default, the last 100 breadcrumbs are kept.
+    */
+        Breadcrumb breadcrumb = new Breadcrumb();
+        breadcrumb.setMessage("User made an action.");
+        Sentry.addBreadcrumb("the user performed an action");
+
+        // Set the user in the current context.
+        Sentry.configureScope(
+            scope -> {
+                User user = new User();
+                user.setEmail("hello@sentry.io");
+                scope.setUser(user);
+            });
+
+    /*
+    This sends a simple event to Sentry, with a simple String message.
+    */
+
+        Sentry.captureMessage("This is a test");
+
+        try {
+            unsafeMethod();
+        } catch (Exception e) {
+        // This sends an error report to the  Sentry
+        Sentry.captureException(e);
```

Comment:
missing a tab here as `Sentry.captureException(e);` is inside of the `catch` block